ui/react-components/components/Conflicts/Body/ConflictsBody.jsx

```diff
@@ -0,0 +1,144 @@
+import {Tab, Tabs, TabList, TabPanel} from 'react-tabs';
+import React, {useState} from "react";
+import "react-tabs/style/react-tabs.css";
+import classNames from 'classnames';
+import PropTypes from "prop-types";
+import {FormattedMessage} from "react-intl";
+import {
+    conflictsCount,
+    conflictsHeading,
+    appointmentConflict,
+    boldContent,
+    conflictMessage,
+    conflictDetails,
+    conflictsList
+} from './ConflictsBody.module.scss';
+import moment from 'moment';
+
+const ConflictsBody = props => {
```

Comment:
Why do we need to have a separate Body folder and have ConflictBody??

Reply:
Just to directory/package organization since there are more components and corresponding specs, css files makes it cumbersome